Makefile

```diff
@@ -13,6 +13,18 @@ MASTER_IMAGE = quay.io/dynatrace/dynatrace-operator:snapshot
 BRANCH_IMAGE = quay.io/dynatrace/dynatrace-operator:snapshot-$(shell git branch --show-current | sed "s/[^a-zA-Z0-9_-]/-/g")
 OLM_IMAGE ?= registry.connect.redhat.com/dynatrace/dynatrace-operator:v${VERSION}
 
+KUBERNETES_OTHERS_YAML=config/deploy/kubernetes/kubernetes-others.yaml
+KUBERNETES_CRD_YAML=config/deploy/kubernetes/kubernetes-crd.yaml
+KUBERNETES_CSIDRIVER_YAML=config/deploy/kubernetes/kubernetes-csidriver.yaml
+KUBERNETES_OLM_YAML=config/deploy/kubernetes/kubernetes-olm.yaml
+KUBERNETES_ALL_YAML=config/deploy/kubernetes/kubernetes-all.yaml
+
+OPENSHIFT_OTHERS_YAML=config/deploy/openshift/openshift-others.yaml
+OPENSHIFT_CRD_YAML=config/deploy/openshift/openshift-crd.yaml
```

Comment:
```suggestion
OPENSHIFT_YAML=config/deploy/openshift/openshift.yaml
```

Reply:
I think @meik99 is right, as CRD makes me think of there is only the CRD included.

Reply:
Actually this was my intention because I overlooked the fact it's actually `CRD`+`OTHERS` merged :see_no_evil:. Thanks for pointing it. 
Anyway, we should call these files different than "kubernetes.yaml" in the future because it means literally nothing.

Reply:
At least for me it makes sense, since `kubernetes.yaml` sounds to me like the default installation file, i would apply to run the operator (everything which is needed to run on kubernetes) and thats what it actually is. I would suggest to keep it as `kubernetes.yaml` and  `openshift.yaml` for now :)